changes/e0bb0e2291cc18ac02c8f5bb385848d3.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Updated ``synapse.tools.genpkg`` to include Synapse version info in the Power-Up
```

Comment:
maybe throw the word "build" in here somewhere since we already have the synapse version in the top-level of the definition